src/test/fs_tests.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2011-2018 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+//
+#include <fs.h>
+#include <test/test_bitcoin.h>
+
+#include <boost/test/unit_test.hpp>
+
+BOOST_FIXTURE_TEST_SUITE(fs_tests, BasicTestingSetup)
+
+BOOST_AUTO_TEST_CASE(fsbridge_fstream)
```

Comment:
In commit "tests: Add test case for std::ios_base::ate" (6b6c598ab55c60ed5050c0ba0a817dba899384c0)

Would it be possible to add a test accessing a utf8 filesystem path now that #13866 is merged?

Reply:
Sure. But this seems only work if #13877 merged, so change this PR to based on it.